src/reactFinalFormContext.js

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+
+export const ReactFinalFormContext = React.createContext({
+  form: null
+})
+
+export const withReactFinalForm = Component => {
+  return class extends React.Component {
+    render() {
+      return React.createElement(ReactFinalFormContext.Consumer, {
```

Comment:
Great question. I've done this before because using JSX introduces, for little benefit, an extra build step / babel plugin, etc. JSX compiles to `React.createElement()` anyway.

Reply:
Sure, but it's used by the project either way & IMHO would make this slightly more readable

Reply:
@erikras Maybe it is time to move to JSX?